staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -295,13 +295,11 @@ var (
 	)
 	apiserverWorkEstimatedSeats = compbasemetrics.NewHistogramVec(
 		&compbasemetrics.HistogramOpts{
-			Namespace: namespace,
-			Subsystem: subsystem,
-			Name:      "work_estimated_seats",
-			Help:      "Number of estimated seats (maximum of initial and final seats) associated with requests in API Priority and Fairness",
-			// the upper bound comes from the maximum number of seats a request
-			// can occupy which is currently set at 10.
-			Buckets:        []float64{1, 2, 4, 10},
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "work_estimated_seats_ratio",
+			Help:           "Ratio of number of estimated seats (maximum of initial and final seats) associated with requests and its maximum value in API Priority and Fairness",
```

Comment:
The word "ratio" begs for two quantities to be identified.  Indeed, speaking of both is needed here.

Reply:
The original `Help` value describes both but I agree it's not the best wording. Changed
> Ratio of number of estimated seats (maximum of initial and final seats) associated with requests and its maximum value in API Priority and Fairness

to
> Ratio of estimated and maximum number of seats associated with requests in API Priority and Fairness